### PR TITLE
add section to set db in pantheon.yml

### PR DESCRIPTION
**Pull request by EdwardAngert**
Should be merged right before PLAT-1915 - sync with @gloria-panth 

## Summary

**[Pantheon](https://pantheon.io/docs/pantheon-yml#specify-a-version-of-mariadb)** - Users can now set the database version for some CMS versions from `pantheon.yml`.

## Remaining Work

The following changes still need to be completed:

- [x] copy edit
- [ ] plat-1915 timing
- [ ] make sure there's no merge conflict in #6145 

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Review by carolyn-shannon — APPROVED** [review]
Looks good.

File: source/content/pantheon-yml.md [hunk 1]
```diff
@@ -117,6 +115,42 @@ php_version: 7.0
...
+
+| CMS            | Recommended MariaDB Version | Configurable After |
+|----------------|-----------------------------|--------------------|
+| Drupal <6.51   | Default                     | May, 2021          |
+| Drupal >= 6.51 | Default                     | May, 2021          |
+| Drupal < 7.76  | Default                     | May, 2021          |
+| Drupal >= 7.76 | Default                     | May, 2021          |
+| Drupal < 8.5   | Default                     | May, 2021          |
+| Drupal >= 8.6  | 10.0 - 10.4                 |                    |
+| Drupal >= 9.0  | 10.4 - 10.4                 |                    |
```
Comment (jburianek) [inline source/content/pantheon-yml.md:151]:
```suggestion
| Drupal >= 8.6  | 10.0 or 10.4                |                    |
| Drupal >= 9.0  | 10.4                        |                    |
```
Specifying a range like `10.0 - 10.4` makes it sound like you can select any version inside that range. Instead, you can right now only choose `10.0` or `10.4`. So we should make that clear.

Also, probably no need to specify a range on Drupal 9.

Comment (EdwardAngert) [reply source/content/pantheon-yml.md]: looks like @joa-pan fixed this one - marking resolved

**Review by joa-pan — APPROVED** [review]
LGTM

@EdwardAngert I made a small change to the table, if you're okay with it I can merge.

**Comment by EdwardAngert** [comment]
@joa-pan - very okay with the change, thank you! We're holding off on merging this until next week while we work out some platform-side requirements

**Pull request merged**